composio/local_tools/local_workspace/cmd_manager/actions/submit_patch.py

```diff
@@ -0,0 +1,55 @@
+from pydantic import Field
+
+from composio.local_tools.local_workspace.commons.get_logger import get_logger
+from composio.local_tools.local_workspace.commons.history_processor import (
+    history_recorder,
+)
+
+from .base_class import BaseAction, BaseRequest, BaseResponse
+
+
+logger = get_logger()
+
+
+class SubmitPatchRequest(BaseRequest):
+    workspace_id: str = Field(
+        ..., description="workspace-id for which patch is generated"
+    )
+    issue_id: str = Field(
+        ...,
+        description="issue-id for which the patch has been generated and needs to be submitted",
+    )
+
+
+class SubmitPatchResponse(BaseResponse):
+    output: str = Field(..., description="output for submit patch")
+
+
+class SubmitPatchCmd(BaseAction):
+    """
+    submits the patch for given issue-id and workspace-id
+    """
+
+    _history_maintains: bool = True
+    _display_name = "submit patch action"
+    _request_schema = SubmitPatchRequest
+    _response_schema = SubmitPatchResponse
+    command = "submit_patch"
+
+    @history_recorder()
+    def execute(
+        self, request_data: SubmitPatchRequest, authorisation_data: dict
+    ) -> BaseResponse:
```

Comment:
This is not the cmdmanager tool. If you want this specifically for SWE agent, then create a different tool benchmark tool or something and then put this tool/action in it. I don't see anywhere else this being used except in those cases.

Reply:
removed